.github/workflows/openapi.yaml

```diff
@@ -24,9 +29,14 @@ jobs:
         run: |
           rm ./trustify-ui/client/openapi/trustd.yaml
           cp ./trustify/openapi.yaml ./trustify-ui/client/openapi/trustd.yaml
-          
           cd ./trustify-ui
           git diff
+
+          if ! git diff --quiet; then
+            npm install 
```

Comment:
Wondering if this shouldn't be `ci` instead of `i`, honoring the lock file?

Reply:
I'm not sure I understand.  Could you rephrase?

Reply:
I think the suggestion is to use the command `npm ci` rather than `npm install` (`npm i` is the shorter command version).  If we copy and paste the command from the README file of the trustify-ui repository the command would be `npm clean-install --ignore-scripts` or `npm ci --ignore-scripts`

Reply:
Yup, thanks for the explanation!

Reply:
ahhh... I had never used `clean-install` before.  Seems like it's what you would want to do for a release build.  But this is not a release build.    It's doing what a developer would do on his machine to submit a PR request.  If that means updating lock files, I think that's ok.  The lock file update will be included in the PR review.

Reply:
:shrug: ok, then :grin: